### PR TITLE
chore: skip highly flaky tests

### PR DESCRIPTION
**Pull request by emilyrohrbough**
This change is to skip a few highly flaky tests

### CODE REVIEW

File: packages/app/cypress/e2e/runs.cy.ts [hunk 1]
```diff
@@ -301,7 +301,11 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
 
   context('Runs - Create Project', () => {
     // TODO: fix flaky test
-    it('when a project is created, injects new projectId into the config file, and sends expected UTM params', { retries: 15 }, () => {
+    it('when a project is created, injects new projectId into the config file, and sends expected UTM params', { retries: 3 }, function () {
+      if (Cypress.config('isTextTerminal') && Cypress.currentRetry > 2) {
+        this.skip()
+      }
+
```
Comment (emilyrohrbough) [inline packages/app/cypress/e2e/runs.cy.ts]:
```suggestion
    it.skip('when a project is created, injects new projectId into the config file, and sends expected UTM params', () => {
```

File: packages/app/cypress/e2e/runs.cy.ts [hunk 2]
```diff
@@ -458,7 +462,11 @@ describe('App: Runs', { viewportWidth: 1200 }, () => {
     })
 
     // TODO: fix flaky test
-    it('opens Connect Project modal after clicking Reconnect Project button', { retries: 15 }, () => {
+    it('opens Connect Project modal after clicking Reconnect Project button', { retries: 3 }, function () {
+      if (Cypress.config('isTextTerminal') && Cypress.currentRetry > 2) {
+        this.skip()
+      }
+
```
Comment (emilyrohrbough) [inline packages/app/cypress/e2e/runs.cy.ts]:
```suggestion
    it.skip('opens Connect Project modal after clicking Reconnect Project button', () => {
```

**Review by jennifer-shehane — APPROVED** [review]
😢

**Review by jordanpowell88 — APPROVED** [review]
#skiptheflake

**Pull request merged**